validator/src/bootstrap.rs

```diff
@@ -484,6 +494,17 @@ fn get_vetted_rpc_nodes(
         let newly_blacklisted_rpc_nodes = RwLock::new(HashSet::new());
         vetted_rpc_nodes.extend(
             rpc_node_details
+                .into_iter()
```

Comment:
Could we move the ping check/sort to the end of the chain? That way we don't waste time/cycles on nodes that got filtered out